src/main/java/com/laytonsmith/core/exceptions/ConfigRuntimeException.java

```diff
@@ -86,32 +86,34 @@ public Environment getEnv() {
 	 * @return
 	 */
 	public static Reaction GetReaction(ConfigRuntimeException e, Environment env) {
-		//If there is an exception handler, call it to see what it says.
-		Reaction reaction = Reaction.REPORT;
+
+		// If there is an exception handler, call it to see what it says.
 		if(env.getEnv(GlobalEnv.class).GetExceptionHandler() != null) {
 			CClosure c = env.getEnv(GlobalEnv.class).GetExceptionHandler();
 			CArray ex = ObjectGenerator.GetGenerator().exception(e, env, Target.UNKNOWN);
 			if(e.getEnv() != null) {
 				MCCommandSender sender = e.getEnv().getEnv(CommandHelperEnvironment.class).GetCommandSender();
 				c.getEnv().getEnv(CommandHelperEnvironment.class).SetCommandSender(sender);
 			}
-			Construct ret = CNull.NULL;
 			try {
 				c.execute(new Construct[]{ex});
+				return Reaction.REPORT; // Closure returned nothing -> REPORT.
 			} catch (FunctionReturnException retException) {
-				ret = retException.getReturn();
-			}
-			if(ret instanceof CNull || Prefs.ScreamErrors()) {
-				reaction = Reaction.REPORT;
-			} else {
-				if(Static.getBoolean(ret, Target.UNKNOWN)) {
-					reaction = Reaction.IGNORE;
+				Construct ret = retException.getReturn();
+				if(ret instanceof CNull || Prefs.ScreamErrors() || !Static.getBoolean(ret, Target.UNKNOWN)) {
+					return Reaction.REPORT; // Closure returned null or false or scream-errors was set in the config.
 				} else {
-					reaction = Reaction.FATAL;
+					return Reaction.IGNORE; // Closure returned true -> IGNORE.
```

Comment:
Hmm, no, we should still set this to FATAL, but wherever it is we handle it, it should be the same behavior